.github/workflows/release-note-changes.yml

```diff
@@ -0,0 +1,58 @@
+name: Release note changes
+
+on:
+  pull_request:
+
+jobs:
```

Comment:
Should we restrict this to run when merging to master?

Reply:
Seeing as we don't do restrict most of our other workflow- probably not. Unless there's a reason you were wanting to move towards restricting workflows to master?

Reply:
I think it makes sense to ask teams to format their changelogs correctly when merging to main, since making sure the CHANGELOG.md is up to date is only a requirement for master.

Also, teams will probably want to reference that merge PR in the changelog itself.

Reply:
Hmm that makes sense to me. And we can always change it in the future super easily.